circle.yml

```diff
@@ -17,17 +17,8 @@ dependencies:
   cache_directories:
     - ~/.kerl
   pre:
-    - |
-      if [[ ! -d ~/.kerl/$OTP_VERSION ]]; then
-        if [[ ! -d ~/.kerl ]]; then
-          mkdir ~/.kerl
-          curl -fsSLo ~/.kerl/kerl https://raw.githubusercontent.com/kerl/kerl/master/kerl
-          chmod +x ~/.kerl/kerl
-          ~/.kerl/kerl update releases
-        fi
-        ~/.kerl/kerl build $OTP_VERSION $OTP_VERSION
-        ~/.kerl/kerl install $OTP_VERSION ~/.kerl/$OTP_VERSION
-      fi
+      - bash ./scripts/circleci-build-erlang.sh:
+          timeout: 3600
```

Comment:
So Erlang takes ~20min to build on Circle.
How about halving this timeout? 30min should be plenty